modules/vm/main.tf

```diff
@@ -123,9 +119,8 @@ resource "azurerm_linux_virtual_machine_scale_set" "graphdb" {
   }
 
   os_disk {
-    # TODO: size? caching?
-    caching              = "None"
-    storage_account_type = "Premium_LRS"
+    caching              = "ReadWrite"
+    storage_account_type = "Standard_LRS"
   }
 
   admin_ssh_key {
```

Comment:
You can replace var.ssh_key with `file(var.ssh_key)` and provide the file instead of the key as string

Reply:
:+1: , I will do this in another PR